Release note landing page and Guides dropdown updated (#5001)
SumoLogic/sumologic-documentation: docs/release-notes/index.md

  <div className="container">
  <a href="/release-notes-cse"><img src={useBaseUrl('img/icons/security/cloud-siem.png')} alt="icon" width="40"/><h4>Cloud SIEM   <img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="40"/></h4></a>
  <p>Information about new and enhanced features, updated content (like rules, log mappers and parsers), bug fixes, and other important announcements for Cloud SIEM.</p>
  </div>
</div>
<div className="box smallbox card">
  <div className="container">
  <a href="/release-notes-csoar"><img src={useBaseUrl('img/icons/security/soar-2-color-icon.png')} alt="icon" width="40"/><h4>Cloud SOAR   <img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="40"/></h4></a>
  <p>The latest news about CSOAR, like new features, bug fixes, changes to the application, and other important announcements for Cloud SOAR.</p>
  </div>
</div>
<div className="box smallbox card">
  <div className="container">
  <a href="/release-notes-collector"><img src={useBaseUrl('img/icons/data-collection.png')} alt="icon" width="40"/><h4>Collector   <img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="40"/></h4></a>

>   <p>Latest features and bug fixes for Installed Collectors. To access new features, upgrade using <a href="/docs/send-data/installed-collectors/collector-installation-reference/download-collector-from-static-url">Static URLs</a>, <a href="/docs/send-data/collection/upgrade-collectors">Sumo Logic</a>, <a href="/docs/send-data/collection/upgrade-collectors">Command Line</a>, <a href="/docs/api/collector-management/upgrade-downgrade-collectors">Collector Management API</a>.</p>

Reviewer: ```suggestion
  <p>Latest features and bug fixes for Installed Collectors. To access new features, upgrade using <a href="/docs/send-data/installed-collectors/collector-installation-reference/download-collector-from-static-url">Static URLs</a>, <a href="/docs/send-data/collection/upgrade-collectors">Sumo Logic</a>, <a href="/docs/send-data/collection/upgrade-collectors">Command Line</a>, or <a href="/docs/api/collector-management/upgrade-downgrade-collectors">Collector Management API</a>.</p>
```

import useBaseUrl from '@docusaurus/useBaseUrl';

> <a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>

Reviewer 2: Can you move this linked RSS icon to the Service Release Notes card? And do the same with the other categories.

> 

Reviewer 2: Could you add an intro at the top, above the cards? Something like _Stay up to date with the latest Sumo Logic enhancements, features, and fixes. Explore the individual release notes pages to see what's new, and consider subscribing to the RSS feeds for automatic updates._

  <p>Latest features and bug fixes for our apps, alerts, security, search, observability, data collectors, and more.</p>
  </div>
</div>
<div className="box smallbox card">
  <div className="container">
  <a href="/release-notes-cse"><img src={useBaseUrl('img/icons/new-releases.png')} alt="icon" width="40"/><h4>Cloud SIEM</h4></a>
  <p>Information about new and enhanced features, updated content (like rules, log mappers and parsers), bug fixes, and other important announcements for Cloud SIEM.</p>
  </div>
</div>
<div className="box smallbox card">
  <div className="container">
  <a href="/release-notes-csoar"><img src={useBaseUrl('img/icons/new-releases.png')} alt="icon" width="40"/><h4>Cloud SOAR</h4></a>
  <p>The latest news about CSOAR, like new features, bug fixes, changes to the application, and other important announcements for Cloud SOAR.</p>
  </div>
</div>
<div className="box smallbox card">
  <div className="container">
  <a href="/release-notes-collector"><img src={useBaseUrl('img/icons/new-releases.png')} alt="icon" width="40"/><h4>Collector</h4></a>

>   <p>New Collector features and relevant bug fixes for each release<sup><a href="#fn1" id="ref1">1</a></sup>.</p>

Reviewer 2: What do you think of this?

```suggestion
  <p>Latest features and bug fixes for Installed Collectors. To access new features, upgrade using <a href="/docs/send-data/installed-collectors/collector-installation-reference/download-collector-from-static-url">Static URLs</a>, <a href="/docs/send-data/collection/upgrade-collectors">Sumo Logic</a>, <a href="/docs/send-data/collection/upgrade-collectors">Command Line</a>, <a href="/docs/api/collector-management/upgrade-downgrade-collectors">Collector Management API</a>.</p>
```

Author: Thanks for this concise information Kim, the card looks better now.

  </div>
</div>
<div className="box smallbox card">
  <div className="container">
  <a href="/release-notes-developer"><img src={useBaseUrl('img/icons/new-releases.png')} alt="icon" width="40"/><h4>Developer</h4></a>
  <p>New features and changes to our APIs, Collector management, and Live Tail CLI.</p>
  </div>
</div>
</div>

> <br>
> </br>
> <div id="footnotes">
>     <ol>
>         <li id="fn1">To use new Collector features mentioned here, you'll need to upgrade your Installed Collector software in one of the following ways:<ul><li> [Using Static URLs](/docs/send-data/installed-collectors/collector-installation-reference/download-collector-from-static-url)</li><li> [Using Sumo Logic](/docs/send-data/collection/upgrade-collectors)</li><li> [Using the Command Line](/docs/send-data/collection/upgrade-collectors)</li><li> [Using Collector Management API](/docs/api/collector-management/upgrade-downgrade-collectors)</li></ul></li>
>     </ol>
> </div>

Reviewer 2: See https://github.com/SumoLogic/sumologic-documentation/pull/5001/files#r1931556182

```suggestion
```
